packages/usa-date-picker/src/index.js

```diff
@@ -2230,12 +2216,6 @@ if (!isIosDevice()) {
     [CALENDAR_DATE_CURRENT_MONTH]() {
       handleMouseoverFromDate(this);
     },
-    [CALENDAR_MONTH]() {
-      handleMouseoverFromMonth(this);
-    },
-    [CALENDAR_YEAR]() {
-      handleMouseoverFromYear(this);
-    },
```

Comment:
Removed unnecessary `handleMousover` functions to prevent unexpected focus on mouseover behaviors